cmd/manager/main.go

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package main
+
+import (
+	"flag"
+	"os"
+	"runtime"
+
+	"github.com/pravega/pravega-operator/pkg/apis"
+	"github.com/pravega/pravega-operator/pkg/controller"
+	"github.com/pravega/pravega-operator/pkg/version"
+
+	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
+	sdkVersion "github.com/operator-framework/operator-sdk/version"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
+
+	log "github.com/sirupsen/logrus"
+)
+
+var (
+	versionFlag bool
+)
+
+func init() {
+	flag.BoolVar(&versionFlag, "version", false, "Show version and quit")
+	flag.Parse()
+}
+
+func printVersion() {
+	log.Printf("pravega-operator Version: %v", version.Version)
+	log.Printf("Git SHA: %s", version.GitSHA)
+	log.Printf("Go Version: %s", runtime.Version())
+	log.Printf("Go OS/Arch: %s/%s", runtime.GOOS, runtime.GOARCH)
+	log.Printf("operator-sdk Version: %v", sdkVersion.Version)
+}
+
+func main() {
+	printVersion()
+	flag.Parse()
+
+	if versionFlag {
+		os.Exit(0)
+	}
+
+	namespace, err := k8sutil.GetWatchNamespace()
+	if err != nil {
+		log.Fatalf("failed to get watch namespace: %v", err)
```

Comment:
Is `Fatalf` a typo?

Reply:
No, `Fatalf` will log the error and exit the program. Ref: https://golang.org/pkg/log/#Fatalf